src/mkContextWrapper.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
+export default function mkContextWrapper(ctx: {}, ctxTypes: {}): React.ComponentType<{ children: () => JSX.Element }> {
```

Comment:
even better, if we don't export `mkContextWrapper` `ctx` can perfectly typed as it's always `{ transition, state }`